platform/ui/src/assets/styles/styles.css

```diff
@@ -1,7 +1,7 @@
 /* CUSTOM OHIF SCROLLBAR */
 .ohif-scrollbar {
   scrollbar-color: #173239 transparent;
-  scrollbar-gutter: stable;
+  overflow-y: auto;
```

Comment:
i'm not sure why you are removing the gutter, it is needed to overlay the thumnail on top of the div, no?